diff --git a/arch/arm/mach-bcm2708/include/mach/platform.h b/arch/arm/mach-bcm2708/include/mach/platform.h
index e063a1e..79ce9d9 100644
--- a/arch/arm/mach-bcm2708/include/mach/platform.h
+++ b/arch/arm/mach-bcm2708/include/mach/platform.h
@@ -219,7 +219,7 @@
 #define PM_RSTS_HADDRF_SET                                 0x00000002
 #define PM_RSTS_HADDRQ_SET                                 0x00000001

-#define UART0_CLOCK      3000000
+#define UART0_CLOCK      12000000

 #endif
